test/e2e/logs/console_decision_logger_test.go

```diff
@@ -89,6 +89,7 @@ func TestDecisionLogWithInput(t *testing.T) {
 		"input":       false,
 		"result":      false,
 		"timestamp":   false,
+		"type":        false,
```

Comment:
[nit] Isn't this redundant? If `type` isn't present, the `t.Fatal` below will stop the test execution. So, this would never trigger an error in the `expectedFields` checks _below that_.

Reply:
Yea ☹️, I think the "right" way to do this would be to have a like optional matcher for each of these expected fields so we can check not only if they exist but if they have the right value in one pass.